src/recovery.rs

```diff
@@ -702,13 +741,18 @@ impl std::fmt::Debug for Recovery {
     }
 }
 
+#[derive(Clone)]
 pub struct Sent {
     pub pkt_num: u64,
 
     pub frames: Vec<frame::Frame>,
 
     pub time_sent: Instant,
 
+    pub time_acked: Option<Instant>,
+
+    pub time_lost: Option<Instant>,
```

Comment:
Same to time_acked. Is it good enough to be bool?

Reply:
This is needed for adaptive thresholds.